src/tools/compiletest/src/header.rs

```diff
@@ -647,6 +654,13 @@ impl Config {
     fn parse_run_rustfix(&self, line: &str) -> bool {
         self.parse_name_directive(line, "run-rustfix")
     }
+
+    fn parse_edition(&self, line: &str) -> Option<String> {
+        match line {
+            ed @ "2015" | "2018" => Some(ed.to_owned()),
```

Comment:
also, maybe `line.trim()`? I don't think we'll have edition names w/ internal whitespace =)